crates/primitives/src/result.rs

```diff
@@ -188,6 +189,13 @@ pub enum InvalidTransaction {
     BlobVersionedHashesNotSupported,
     /// Block `blob_gas_price` is greater than tx-specified `max_fee_per_blob_gas` after Cancun.
     BlobGasPriceGreaterThanMax,
+    /// There should be at least one blob in Blob transaction.
+    EmptyBlobs,
+    /// Blob transaction can't be a create transaction.
+    /// `to` must be present
+    BlobCreateTransaction,
+    /// Transaction has more then [`crate::MAX_BLOB_NUMBER_PER_BLOCK`] blobs
+    TooManyBlobs,
 }
```

Comment:
```suggestion
    /// Blob transaction contains a versioned hash with an incorrect version 
    BlobMismatchedVersion,
}
```

Reply:
BlobMismatchedVersion is related to precompile. I sow that `BlobVersionedHashesNotSupported` was not used, probably we missed it so I used it to report not supported blob hash version

Reply:
Ah I see, if unused elsewhere then maybe it could just be renamed. my other comment assumes it was used

Reply:
I apologize this was not correct, will add additional field

Reply:
Renamed it as `BlobVersionNotSupported`